test/suites/clustering.sh

```diff
@@ -2763,7 +2763,7 @@ test_clustering_image_refresh() {
   LXD_DIR="${LXD_REMOTE_DIR}" lxc config set core.https_address "10.1.1.104:8443"
 
   # Add remotes
-  lxc remote add public "https://10.1.1.104:8443" --accept-certificate --token foo --public
+  lxc remote add public "https://10.1.1.104:8443" --accept-certificate --public
```

Comment:
Can we add tests for not being able to use --accept-certificate with tokens please?